cli/cmd/status/status.go

```diff
@@ -128,13 +126,6 @@ func (c *Command) Run(args []string) int {
 		c.UI.Output(s, terminal.WithSuccessStyle())
 	}
 
-	if s, err := c.checkConsulClients(namespace); err != nil {
-		c.UI.Output(err.Error(), terminal.WithErrorStyle())
-		return 1
-	} else {
-		c.UI.Output(s, terminal.WithSuccessStyle())
-	}
-
```

Comment:
Should we instead check if clients are enabled and only check status then? It's also a bit weird that we check servers all the time even for clusters that might not have servers installed.

Reply:
Yeah, that's possible. We can have it read the deployment configuration and determine what to check then. That would be a separate set of work. Should I close this PR and take that approach instead?

Reply:
Ok, sounds good. I'll approve this one - I think it's good to get at least that merged in.

Reply:
https://hashicorp.atlassian.net/browse/NET-1037